src/Core/src/Platform/Android/LayoutChangedEventArgs.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace Microsoft.Maui.Platform
+{
+	public class LayoutChangedEventArgs : EventArgs
```

Comment:
Sadly this type is public but was only ever used for internal things. I moved it to a new file so I never have to see it again.

Reply:
Even if we won't use it, adding a doc comment "For internal use by the <namespace>/platform" can help out